### PR TITLE
Don't offer free tier link if default KC patch (only) is detected.

### PR DESCRIPTION
**Pull request by estrabd**
Case CPANEL-17487: If the default KernelCare patch set is detected, we
do not want to offer an automated upgrade to the free tier because
this will currently undo the default patch set in favor of the free.

Work done for SWAT-780.

### CODE REVIEW

File: pkg/Cpanel/Security/Advisor/Assessors/Apache.pm [hunk 1]
```diff
@@ -256,17 +256,16 @@ sub _centos_symlink_protection {
...
                 'key'        => 'Apache_no_symlink_protection',
                 'type'       => $bad,
-                'text'       => $self->_lh->maketext('No symlink protection detected'),
-                'suggestion' => $self->_lh->maketext(
-                    'You do not appear to have any symlink protection enabled on this server. You can protect against this in multiple ways. Please review the following [output,url,_1,documentation,_2,_3] to find a solution that is suited to your needs.',
-                    ($is_ea4) ? 'https://go.cpanel.net/EA4Symlink' : 'https://go.cpanel.net/apachesymlink',
-                    'target',
-                    '_blank'
-                ),
+                'text'       => q{Kernel does not support the prevention of symlink ownership attacks.},
```
Comment (bk2204) [inline pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: This text doesn't seem to be localized, but it probably should be.

File: pkg/Cpanel/Security/Advisor/Assessors/Apache.pm [hunk 2]
```diff
@@ -256,17 +256,16 @@ sub _centos_symlink_protection {
...
+                'suggestion' => $doclink,
```
Comment (bk2204) [inline pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: I don't have a strong opinion either way, but is there a reason that you split this out above?  This is different from what we do elsewhere and it does make review a bit more difficult.

Comment (estrabd) [reply pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: No reason, it was a game time decision. I see it's inconsistent, but if we're not going to localize this change isn't worth our time if we're not going localize the string above. Please consider accepting this PR as is.

File: pkg/Cpanel/Security/Advisor/Assessors/Apache.pm (hunk 1 above)
Comment (estrabd) [reply pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: brian, Kelli said we've never consistently localized Security Advisor and that should be addressed as technical debt separately. Please reconsider.

Comment (bk2204) [reply pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: That's true, but in this case it's as easy as adding a call to maketext.  Customers used to be able to localize this UI if they wanted, and we shouldn't regress that.

File: pkg/Cpanel/Security/Advisor/Assessors/Apache.pm (hunk 2 above)
Comment (bk2204) [reply pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: As I said, I don't have a strong opinion, so you needn't fix this issue if you don't want to.

File: pkg/Cpanel/Security/Advisor/Assessors/Apache.pm (hunk 1 above)
Comment (estrabd) [reply pkg/Cpanel/Security/Advisor/Assessors/Apache.pm]: I see. Thank you.

**Comment by estrabd** [comment]
@bk2204 ready for review again. Thanks!

**Comment by bk2204** [comment]
This change looks fine except that you still have some failing tests.  Could you fix those, please?

**Pull request merged**